src/main/java/eu/tsystems/mms/tic/testerra/plugins/xray/mapper/xray/XrayTestExecutionImport.java

```diff
@@ -414,17 +435,17 @@ public XrayTestExecutionImport(XrayTestExecutionIssue testExecutionIssue) {
         this.info.setSummary(testExecutionIssue.getSummary());
         this.info.version = testExecutionIssue.getFixVersions().stream().findFirst().map(JiraNameReference::getName).orElse(null);
         this.info.revision = testExecutionIssue.getRevision();
-        this.info.user = testExecutionIssue.getAssignee().getName();
+        this.info.user = testExecutionIssue.getAssignee() != null ? testExecutionIssue.getAssignee().getName() : null;
```

Comment:
`JiraIssue#getAssignee` calls `JiraIssue#getOrCreateEntity` and cannot return `null`.

Reply:
agree, fixing now